apitesters/enums.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  ATTRIBUTION_NETWORK,
+  BILLING_FEATURE,
+  PURCHASE_TYPE,
+  PACKAGE_TYPE,
+  INTRO_ELIGIBILITY_STATUS,
+  PRORATION_MODE
+} from '../dist';
+
+function checkAttributionNetwork(network: ATTRIBUTION_NETWORK): boolean {
+    switch (network) {
+        case ATTRIBUTION_NETWORK.APPLE_SEARCH_ADS: return true;
+        case ATTRIBUTION_NETWORK.ADJUST: return true;
+        case ATTRIBUTION_NETWORK.APPSFLYER: return true;
+        case ATTRIBUTION_NETWORK.BRANCH: return true;
+        case ATTRIBUTION_NETWORK.TENJIN: return true;
+        case ATTRIBUTION_NETWORK.FACEBOOK: return true;
```

Comment:
No idea but are typescript switch-cases exhaustive? As in, would it fail to compile if we are missing a `case`? I'm guessing that's the reason for the boolean return, so it fails with a missing return if a case is missing, but just wanted to check.

Reply:
Yes! I forgot how I got this to work. Because of the `noImplicitReturns` rule, each branch must return something. That ensure there aren't missing cases (that's why these return a meaningless Boolean)